source/2020-05-08-the-ember-times-issue-147.md

```diff
@@ -0,0 +1,152 @@
+
+- inspection of **component arguments**: You can now see and investigate [arguments](https://emberjs.github.io/rfcs/0311-angle-bracket-invocation.html#arguments) passed to your components through Ember Inspector's component tree 🌳
+- inspection of **the store**: Directly inspect your data store and send it as a variable for further debugging to your console. You can check out a demo recording of this feature in the [PR description](https://github.com/emberjs/ember-inspector/pull/1163) of [Sarah Yu's (@SYU15)](https://github.com/SYU15) work 🏬
+- navigate through the component tree via **keyboard** with a new [accessibility improvement](https://github.com/emberjs/ember-inspector/pull/1153) 🎹
+
+Check out the full list of features and improvements over at the project's [changelog](https://github.com/emberjs/ember-inspector/blob/v4.0.1/CHANGELOG.md#v400-2020-05-06). A sparkling thank you goes to the contributors [Sarah Yu (@SYU15)](https://github.com/SYU15), [Jerry Nummi (@nummi)](https://github.com/nummi), [Robert Wagner (@rwwagner90)](https://github.com/rwwagner90) and [Godfrey Chan (@chancancode)](https://github.com/chancancode) who made this release happen!
+
+---
+
+## [Blog Post: Comparing Ember Octane and React 🐹⚛️](https://www.pzuraq.com/comparing-ember-octane-and-react/)
+
+[Chris Garrett (@pzuraq)](https://github.com/pzuraq) wrote a blog post [comparing Ember Octane and React](https://www.pzuraq.com/comparing-ember-octane-and-react/) in terms of their respective programming models and developer experience using the latest idioms and best practices from both frameworks (Ember Octane and React Hooks).
+
+The blog post goes through an in-depth analysis of the programming models between Ember and React including how they handle derived values and state, lifecycle hooks or methods to handle scheduling, and handling user interaction.
+
+Check out the [takeaways](https://www.pzuraq.com/comparing-ember-octane-and-react/) from this comparison in the full blog post!
+
+---
+
+## [Auto save en Ember Octane 💾](https://betocantu93.com/autosave-in-ember-octane/)
+
+[Alberto Cantú (@betocantu93)](https://github.com/betocantu93), in his first blog post in Spanish, explores [how to implement autosave in Ember](https://betocantu93.com/autosave-in-ember-octane/). We encourage you to check it out! Alberto starts with a basic form with a submit button, then introduces autosave and addresses potential problems using `ember-concurrency`.
+
+Do you follow a similar pattern or have another approach to autosaving your user's form? Share your ideas with the JS community today!
+
+---
+
+## [Embroider & memoization RFCs entered final comment period 💬](https://github.com/emberjs/rfcs/pull/507)
+
+In case you missed, the RFCs for [Embroider](https://github.com/emberjs/rfcs/pull/507) and [memoizing tracked properties](https://github.com/emberjs/rfcs/pull/615) entered their final comment period (FCP) on May 1.
```

Comment:
I corrected what is memoized (didn't read the RFC carefully).

```suggestion
In case you missed, the RFCs for [Embroider](https://github.com/emberjs/rfcs/pull/507) and [memoizing functions](https://github.com/emberjs/rfcs/pull/615) entered their final comment period (FCP) on May 1.
```